test/e2e/playwright.config.ts

```diff
@@ -54,6 +55,30 @@ const config: PlaywrightTestConfig = {
 		{
 			name: 'chromium',
 			use: { ...devices[ 'Desktop Chrome' ] },
+			grepInvert: /-chromium/,
+		},
+		{
+			name: 'webkit',
+			use: {
+				...devices[ 'Desktop Safari' ],
```

Comment:
So this isn't exactly Safari, but a Webkit based browser? Interesting.

Reply:
Yeah, it's the same as running in Chromium instead of Chrome or Edge. Another reason is that Safari is only available on macOS, but webkit is also available on Linux and Windows.